docs/en/04_Performance_Guide/02_HTTP_Caching.md

```diff
@@ -74,16 +82,18 @@ This means the response is classified as the "None" cache level.
-These responses are effectively the "Full" cache level. The `max-age` value is currently 120 seconds, but could change in the future. CWP customers can’t actively clear CDN caches on Incapsula unless they purchase an optional Premium Managed Service plan. Due to this restriction, asset invalidation needs to take place via the URL, through so called “cache busters”. SilverStripe adds a GET parameter with the last file modification timestamp to each stylesheet and javascript file included through its [Requirements API](http://docs.silverstripe.org/en/3/developer_guides/templates/requirements/). If you are referencing files in other ways, please take care to add your own “cache busters”, e.g. through a Grunt build task modifying the including SilverStripe template.
+Furthermore, all CWP instances are configured to set the following header on anything that is NOT served by the framework. This includes all requests for theme files and any asset requests that are not served by the [Secure Assets module](https://github.com/silverstripe/silverstripe-secureassets).
+These responses are effectively the "Full" cache level. The `max-age` value is currently 120 seconds, but could change in the future. CWP customers can’t actively clear CDN caches on Incapsula unless they purchase an [optional Premium Managed Service plan](https://www.cwp.govt.nz/features/optional-extras/). 
+Due to this restriction, asset invalidation needs to take place via the URL, through so called “cache busters”. SilverStripe adds a GET parameter with the last file modification timestamp to each stylesheet and javascript file included through its [Requirements API](http://docs.silverstripe.org/en/4/developer_guides/templates/requirements/). 
+If you are referencing files in other ways, please take care to add your own “cache busters”, e.g. through a Grunt build task modifying the including SilverStripe template.
 
 ```
 Cache-Control: max-age=120, public
 ```
 
 #### Light caching on dynamic content
 
-You'll need to talk to your business owner about cache lifetimes: Updated content might not reach visitors until caches expire. This policy is generally safe, but specific controllers may need tweaks. For example the "userforms" module requires caching to be disabled through a `NoopPolicy` because it generates a unique form submission token for each visitor.
+You'll need to talk to your business owner about cache lifetimes: Updated content might not reach visitors until caches expire. This policy is generally safe, but specific controllers may need tweaks. For example, the [UserForms module](https://github.com/silverstripe/silverstripe-userforms) requires caching to be disabled through a `NoopPolicy` because it generates a unique form submission token for each visitor.
```

Comment:
>requires caching to be disabled through a `NoopPolicy`

Change to something like "requires caching to be disabled by removing any controllerpolicy module cache policies applied to it with YAML configuration"